jessie_update/README.md

```diff
@@ -3,21 +3,21 @@
 This directory contains the scripts we used to upgrade Jessie to Raspbian for Robots.  This is simply here for development purposes and so that it might be found to be useful to others.
 
 # To Begin
-+From the home directory run `sudo wget https://raw.githubusercontent.com/DexterInd/Raspbian_For_Robots/master/jessie_update/regimen.sh`
++ From the home directory run `sudo wget https://raw.githubusercontent.com/DexterInd/Raspbian_For_Robots/master/jessie_update/regimen.sh`
 
-+From the home directory run `sudo sh regimen.sh`
++ From the home directory run `sudo sh regimen.sh`
```

Comment:
No longer required. The libraries are being removed in cleanup anyway. With the use of PiClone, this step can be skipped